AdyenActions/UI/View Controllers/Document/DocumentActionView.swift

```diff
@@ -0,0 +1,102 @@
+        imageView.accessibilityIdentifier = ViewIdentifierBuilder.build(scopeInstance: self, postfix: "icon")
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        imageView.widthAnchor.constraint(equalToConstant: 80).isActive = true
+        imageView.heightAnchor.constraint(equalToConstant: 52).isActive = true
+        imageView.adyen.round(using: style.image.cornerRounding)
+        imageView.contentMode = style.image.contentMode
+        imageView.layer.masksToBounds = style.image.clipsToBounds
+        imageView.layer.borderColor = style.image.borderColor?.cgColor
+        imageView.layer.borderWidth = style.image.borderWidth
+        imageView.backgroundColor = style.image.backgroundColor
+        
+        return imageView
+    }()
+    
+    internal lazy var messageLabel: UILabel = {
+        let label = UILabel(style: style.messageLabel)
+        label.text = viewModel.message
+        label.numberOfLines = 0
+        label.lineBreakMode = .byWordWrapping
+        label.setContentHuggingPriority(.required, for: .vertical)
+        label.accessibilityIdentifier = ViewIdentifierBuilder.build(scopeInstance: self, postfix: "messageLabel")
+        
+        return label
+    }()
+    
+    private lazy var mainButton: UIButton = {
+        let button = UIButton(style: style.mainButton)
+        button.translatesAutoresizingMaskIntoConstraints = false
+        button.setTitle(viewModel.buttonTitle, for: .normal)
+        button.addTarget(self, action: #selector(onMainButtonTap), for: .touchUpInside)
+        button.accessibilityIdentifier = ViewIdentifierBuilder.build(scopeInstance: self, postfix: "mainButton")
+        button.preservesSuperviewLayoutMargins = true
+        button.heightAnchor.constraint(equalToConstant: 50.0).isActive = true
+        
+        return button
+    }()
+
+    internal weak var delegate: ActionViewDelegate?
+    
+    /// The view model.
+    private let viewModel: DocumentActionViewModel
+    
+    /// The UI style.
+    private let style: DocumentComponentStyle
+    
+    internal init(viewModel: DocumentActionViewModel, style: DocumentComponentStyle) {
+        self.viewModel = viewModel
+        self.style = style
+        super.init(frame: .zero)
+        configureViews()
+    }
+    
+    @available(*, unavailable)
+    internal required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    private func configureViews() {
+        backgroundColor = style.backgroundColor
+        addSubview(stackView)
+        stackView.adyen.anchor(inside: .view(self), edgeInsets: .init(left: 20, right: -20))
+        NSLayoutConstraint.activate([stackView.centerYAnchor.constraint(equalTo: centerYAnchor),
+                                     stackView.topAnchor.constraint(greaterThanOrEqualTo: topAnchor),
+                                     stackView.bottomAnchor.constraint(lessThanOrEqualTo: bottomAnchor, constant: -30)])
+        
```

Comment:
```swift
stackView.centerYAnchor.constraint(equalTo: centerYAnchor),
stackView.topAnchor.constraint(greaterThanOrEqualTo: topAnchor), // <-- this one is explicit
stackView.bottomAnchor.constraint(lessThanOrEqualTo: bottomAnchor, constant: -30)
```

Reply:
Couldnt get what you mean here

Reply:
I meant "excessive" 😅

Reply:
Since it is not strict, I think it's fine to keep it right?